code/modules/power/supermatter/supermatter.dm

```diff
@@ -168,7 +168,8 @@ GLOBAL_DATUM(main_supermatter_engine, /obj/machinery/power/supermatter_shard)
 	for(var/mob/living/carbon/human/H in viewers(range, src))
 		if(H != user)
 			continue
-		if(!istype(H.glasses, /obj/item/clothing/glasses/meson))
+		var/obj/item/organ/eyes/eyes = H.getorganslot("eye_sight")
+		if(!istype(H.glasses, /obj/item/clothing/glasses/meson) && !istype(eyes, /obj/item/organ/eyes/robotic))
```

Comment:
do

```
if(istype(H.glasses, /obj/item/clothing/glasses/meson))
    continue
var/obj/item/organ/eyes/eyes = H.getorganslot("eye_sight")
if(eyes && eyes.status != ORGAN_ROBOTIC)
    continue```

Reply:
I tried to make this block of code work, but I couldn't. Where does the `to_chat(H, "<span class='danger'>You get headaches just from looking at it.</span>")` go? Replace the second continue maybe?